cluster-commands_test.go

```diff
@@ -0,0 +1,52 @@
+package madmin
```

Comment:
```suggestion
package madmin
```

preserve the copyright @taran-p

Reply:
whoops, fixed